app/controllers/users.js

```diff
@@ -15,3 +18,19 @@ exports.signUp = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.signIn = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    const user = await findUserByEmail(email);
```

Comment:
You can maybe use an interactor here to call your service outside the controller, but is not wrong either.

Reply:
I'll keep it in mind. thanks